plugins/modules/vmware_guest_controller.py

```diff
@@ -437,13 +450,17 @@ def configure_disk_controllers(self):
                     if usb_exists:
                         self.module.warn("'%s' USB controller already exists, can not add more." % disk_ctl_config['type'])
                     else:
-                        disk_controller_new = self.create_controller(disk_ctl_config['type'])
+                        disk_controller_new = self.create_controller(disk_ctl_config['type'], disk_ctl_config.get('bus_sharing'))
                         self.config_spec.deviceChange.append(disk_controller_new)
                         self.change_detected = True
                 # create other disk controller
                 else:
                     if disk_ctl_config.get('controller_number') is not None:
-                        disk_controller_new = self.create_controller(disk_ctl_config['type'], disk_ctl_config.get('controller_number'))
+                        disk_controller_new = self.create_controller(
+                                disk_ctl_config['type'],
+                                disk_ctl_config.get('bus_sharing'),
+                                disk_ctl_config.get('controller_number')
```

Comment:
```suggestion
                            disk_ctl_config['type'],
                            disk_ctl_config.get('bus_sharing'),
                            disk_ctl_config.get('controller_number')
```

The CI is still running but I've already seen that it will fail with a PEP8 failure: `E126: continuation line over-indented for hanging indent`. Removing 4 spaces should fix this.

Reply:
fixed in 1a37832